src/main/java/com/fabriccommunity/spookytime/SpookyTime.java

```diff
@@ -4,10 +4,15 @@
 import com.fabriccommunity.spookytime.common.SpookyItems;
 import com.fabriccommunity.spookytime.common.SpookyEntities;
 import net.fabricmc.api.ModInitializer;
+import net.minecraft.util.Identifier;
 
-public class SpookyTime implements ModInitializer
+public final class SpookyTime implements ModInitializer
```

Comment:
Not a fan of final classes, but I suppose this is fine. Not like people are gonna extend a mod initializer.